compiler/rustc_borrowck/src/region_infer/mod.rs

```diff
@@ -132,7 +132,7 @@ impl RegionTracker {
 
     /// Returns `true` if during the annotated SCC reaches a placeholder
     /// with a universe larger than the smallest reachable one, `false` otherwise.
-    pub fn has_incompatible_universes(&self) -> bool {
+    pub(crate) fn has_incompatible_universes(&self) -> bool {
         self.universe().cannot_name(self.max_placeholder_universe_reached)
```

Comment:
```suggestion
        self.min_universe().cannot_name(self.max_placeholder_universe_reached)
```

This name would have been clearer to me. I was paranoid and had to go find the definition.